docker/ResourceManager.dockerfile

```diff
@@ -0,0 +1,33 @@
+# Myriad - YARN Resource Manager
+#
+# VERSION 0.0.1
+
+FROM centos
+MAINTAINER Brandon Gulla <brandon@brandongulla.com>
```

Comment:
Do you want to be sole maintainer? For some Mesos things, we use "Apache Mesos <dev@mesos.apache.org>" so that users email the mailing list when they have questions, rather than a single person who may not be available.